src/controllers/controller.cpp

```diff
@@ -108,14 +108,19 @@ void Controller::receive(const QByteArray data) {
     int length = data.size();
     if (debugging()) {
         // Formatted packet display
-        QString message = QString("%1: %2 bytes:\n").arg(m_sDeviceName).arg(length);
-        for(int i=0; i<length; i++) {
-            QString spacer=" ";
-            if ((i+1) % 4 == 0) spacer="  ";
-            if ((i+1) % 16 == 0) spacer="\n";
-            message += QString("%1%2")
-                        .arg((unsigned char)(data.at(i)), 2, 16, QChar('0')).toUpper()
-                        .arg(spacer);
+        QString message =
+                QString("%1: %2 bytes:\n").arg(m_sDeviceName).arg(length);
+        for (int i = 0; i < length; i++) {
+            QString spacer = " ";
+            if ((i + 1) % 4 == 0)
+                spacer = "  ";
+            if ((i + 1) % 16 == 0)
+                spacer = "\n";
+            message +=
+                    QString("%1%2")
+                            .arg((unsigned char)(data.at(i)), 2, 16, QChar('0'))
+                            .toUpper()
+                            .arg(spacer);
```

Comment:
This is also a question on line length.
I prefer to allow longer lines. 

The core issue is that in my experience you find always cases where a auto-formater fails. 
I'm my real world project we use 120 coloumns and we have similar issues. 

And that is also the main argument, because I dislike line breaks introduced by auto formater.
They are simply not able to break there own rules, following a master rule: "Improve readability" 

There seams to be a strong urge to auto-reformat the code. How about setting the line length to unlimited. For my feeling we have no readability issues with long lines inside the Mixxx code base. I think than my biggest objection is solved. This way all the minor annoyances are gone without introducing to much new ones.